javascript-light-bulb/main.js

```diff
@@ -0,0 +1,13 @@
+var lightBulb = document.querySelector('.light-bulb');
+var container = document.querySelector('.container');
+lightBulb.addEventListener('click', flipLightSwitch);
+
+function flipLightSwitch() {
+  if (lightBulb.className === 'light-bulb') {
```

Comment:
This falls under what we discussed yesterday, about how we do not want information on the DOM to be our only source of data we need to do work in Javascript. In this case, rather than deciding your work to be done based on the state of the elements in the DOM, you would be better off having a variable that keeps track of what state the DOM is currently in i.e. whether the lightbulb is on or off.